packages/apps/docs/src/components/Markdown/Code/Code.tsx

```diff
@@ -1,15 +1,55 @@
-import { StyledCode, StyledInlineCode } from './styles';
+import { code, codeLine, codeWord, inlineCode } from './style.css';
 
-import React, { FC, ReactNode } from 'react';
+import { useTheme } from '@/hooks';
+import React, { FC, ReactNode, useEffect } from 'react';
 
 interface IProp {
   children: ReactNode;
 }
 
 export const Code: FC<IProp> = ({ children, ...props }) => {
+  const { theme } = useTheme();
+
+  const [themedProps, setThemedProps] = React.useState(props);
+
+  useEffect(() => {
+    setThemedProps({
+      ...props,
+      'data-active-theme': theme,
+    });
+  }, [props, theme]);
+
   if (typeof children === 'string') {
-    return <StyledInlineCode>{children}</StyledInlineCode>;
+    return <code className={inlineCode}>{children}</code>;
   }
 
-  return <StyledCode {...props}>{children}</StyledCode>;
+  return (
+    <code className={code} {...themedProps}>
+      {React.Children.map(children, (child) => {
+        if (!React.isValidElement(child) || !child) {
+          return null;
+        }
+
+        const grandchildren = React.Children.map(
+          child.props.children,
+          (grandchild: ReactNode) => {
+            if (!React.isValidElement(grandchild) || !grandchild) {
+              return null;
+            }
+
+            return React.cloneElement(grandchild, {
+              ...grandchild.props,
+              className: codeWord,
+            });
+          },
```

Comment:
I'm confused by what's happening here. Can you explain the context?

Reply:
The stitches code used nested styles, which we can't really do with sprinkles:
https://github.com/kadena-community/kadena.js/blob/acfba63a66039844edf8972a235310c40029956e/packages/apps/docs/src/components/Markdown/Code/styles.ts#L9

https://github.com/kadena-community/kadena.js/blob/acfba63a66039844edf8972a235310c40029956e/packages/apps/docs/src/components/Markdown/Code/styles.ts#L106

This nested recursion over children and grandchildren is to assign the `codeLine` and `codeWord` respectively.

Reply:
ahh ok. If that is the only reason for this, I think this is an instance where using `globalStyle` would be worth doing considering it's much simpler and easier to maintain/understand

Reply:
Tried using `globalStyle` but for some reason the styles aren't being applied 

Implemented the `grandchildren` portion using globalStyle in the last push, but if visiting a code portion (ie http://localhost:3000/docs/pact/beginner/hello-world) doesn't show the code in the `$mono` font